Explain OpenCodelists organisations (#1453)
opensafely/documentation: docs/codelist-creation.md

OPCS-4 and dm+d codes are not currently supported by the OpenCodelists builder. However, it is possible to [manually upload an existing OPCS-4 or dm+d codelist](https://www.opencodelists.org/docs/#creating-a-codelist-from-a-csv-file).

## Organisations

> Codelists on OpenCodelists can be owned by individuals or by organisations. Creating codelists under an organisation means that the codelist easier to find and reuse by both members of your organisation and those in other organisations. Only codelists created by organisations are included in search results on OpenCodelists.

Reviewer: ```suggestion
Codelists on OpenCodelists can be owned by individuals or by organisations. Creating codelists under an organisation means that the codelist is easier to find and reuse. Only codelists created by organisations are included in search results on OpenCodelists.
```
